packages/@aws-cdk/aws-glue-alpha/test/integ.job-python-shell.js.snapshot/aws-glue-job-python-shell.template.json

```diff
@@ -1,6 +1,6 @@
 {
  "Resources": {
-  "ShellJobServiceRoleCF97BC4B": {
+  "IAMServiceRole61C662C4": {
```

Comment:
Is it supposed to be changed to IAMServiceRole here from ShellJob... specifically?

Reply:
This is based on resource name in the new integration tests. The value is passed in.